modules/systemSettings.js

```diff
@@ -0,0 +1,321 @@
+function SystemSettings() {
+
+  this.systemsettings = {
```

Comment:
Still not super comfortable enforcing defaults for these. @nlarsonX, is it possible to read the current value of the settings in question without having a dummy value?